app/serializers/api/v1/stockit_item_serializer.rb

```diff
@@ -31,6 +32,30 @@ def include_order?
 
     alias_method :include_designation_id?, :include_order?
 
+    def include_added_quantity?
+      @options[:include_added_quantity]
+    end
+
+    def include_in_hand_quantity?
+      @options[:include_in_hand_quantity]
+    end
+
+    def in_hand_quantity
+      object.total_in_hand_quantity
+    end
+
+    def in_hand_quantity__sql
+      "select sum(quantity) from packages_inventories where package_id = #{object.id}"
+    end
+
+    def added_quantity
```

Comment:
The way this `added_quantity` is used doesn't make a lot of sense.

It defines "the quantity which is in all of the boxes"
But according to the Stock PR, it is used to "remove that quantity from a single box"

I think the frontend, for a given package, should fetch the list of boxes and quantities. And use that to trigger pack/unpack commands. Having a single property on the serializer seems faulty to me

Reply:
sure, will check and fix that.